docs/Changelog.md

```diff
@@ -15167,6 +15167,8 @@ This version of the operator has been available since version 13 of the default
   Clip operator limits the given input within an interval. The interval is
   specified by the inputs 'min' and 'max'. They default to
   numeric_limits::lowest() and numeric_limits::max(), respectively.
+  When 'min' is greater than 'max', the clip operator sets all the 'input' values to
+  the value of 'max'.
```

Comment:
nit: maybe adding the description from numpy would be helpful: that is, adding `Thus, this is equivalent to Min(max, Max(input, min)`. As I started typing this, I realized that reuse of min/max as names of inputs actually makes it complicated to write this! I ended up using `Min/Max` to indicated the corresponding functions, but it can be a bit confusing too.

Reply:
I see what you mean, this additional sentence would still be helpful in my opinion, will add.